packages/katello/katello/katello-remove

```diff
@@ -68,43 +74,43 @@ LOG_FILES = [
 ]
 
 RPMS = [
-  'puppetlabs-release',
-  'foreman-release',
-  'foreman-client',
-  'foreman-proxy',
-  'candlepin',
-  'katello',
+  '$HOSTNAME',
```

Comment:
Does this work? By default variables within single quotes are not interpolated

Reply:
@ekohl it works, it looks like it has been that way for a while, I was surprised too. I did a quick test on up/downstream to confirm. Here are the findings showing it works:

Before on upstream:
```bash
[root@centos7-katello-nightly ~]# rpm -qa | grep nightly
centos7-katello-nightly.area51.example.com-qpid-router-server-1.0-1.noarch
katello-repos-3.13.0-0.1.nightly.el7.noarch
centos7-katello-nightly.area51.example.com-tomcat-1.0-1.noarch
centos7-katello-nightly.area51.example.com-foreman-proxy-1.0-1.noarch
centos7-katello-nightly.area51.example.com-puppet-client-1.0-1.noarch
centos7-katello-nightly.area51.example.com-qpid-broker-1.0-1.noarch
centos7-katello-nightly.area51.example.com-apache-1.0-1.noarch
centos7-katello-nightly.area51.example.com-foreman-proxy-client-1.0-1.noarch
centos7-katello-nightly.area51.example.com-qpid-client-cert-1.0-1.noarch
centos7-katello-nightly.area51.example.com-foreman-client-1.0-1.noarch
centos7-katello-nightly.area51.example.com-qpid-router-client-1.0-1.noarch
```
After on upstream:
```bash
[root@centos7-katello-nightly sbin]# updatedb
[root@centos7-katello-nightly sbin]# rpm -qa | grep nightly
```
Before on downstream:
```bash
[root@sat-6-6-qa-rhel7 ~]# rpm -qa | grep area51
sat-6-6-qa-rhel7.area51.example.com-qpid-broker-1.0-1.noarch
sat-6-6-qa-rhel7.area51.example.com-apache-1.0-1.noarch
sat-6-6-qa-rhel7.area51.example.com-foreman-proxy-1.0-1.noarch
sat-6-6-qa-rhel7.area51.example.com-qpid-router-client-1.0-1.noarch
sat-6-6-qa-rhel7.area51.example.com-qpid-client-cert-1.0-1.noarch
sat-6-6-qa-rhel7.area51.example.com-foreman-client-1.0-1.noarch
sat-6-6-qa-rhel7.area51.example.com-qpid-router-server-1.0-1.noarch
sat-6-6-qa-rhel7.area51.example.com-foreman-proxy-client-1.0-1.noarch
sat-6-6-qa-rhel7.area51.example.com-tomcat-1.0-1.noarch
sat-6-6-qa-rhel7.area51.example.com-puppet-client-1.0-1.noarch
```
After on downstream:
```bash
[root@sat-6-6-qa-rhel7 ~]# updatedb
[root@sat-6-6-qa-rhel7 ~]# rpm -qa | grep area51
```